tools/slack/guidelines.md

```diff
@@ -73,6 +73,10 @@ Slack Groups allow you to direct messages to a specific list of people in a more
 
 Groups are a great way to alert people who might not be in a channel about something that needs their attention or make sure urgent incidents are directed to the right people and not an entire channel. For example, you can ping **@github-admins** in #admins-github if you have an urgent issue instead of using an **@channel** or use **@federalist-team** to alert that group of a conversation in another channel that we should see immediately.
 
+## Add yourself to [our custom emoji](https://gsa-tts.slack.com/customize/emoji)
```

Comment:
I wouldn't even include this here either and just let people discover it when they read the [guide](https://docs.google.com/document/d/1Hm42cg61S7FPhaLrRIJxl-LXQCcwGvJTKX_wG0Jz4aU/edit#heading=h.ho6etfp5g3pp) that is linked at the top

Reply:
Not seeing where it talks about adding emoji, but anyway, let's address separately: https://github.com/18F/handbook/issues/2196